.github/workflows/versioned-security-agent.yml

```diff
@@ -28,7 +28,7 @@ jobs:
 
     strategy:
       matrix:
-        node-version: [14.x, 16.x, 18.x]
+        node-version: [16.x, 18.x]
```

Comment:
I must've missed adding 20.x, can you do that please?

Reply:
Given that this was the security agent, I'd thought that'd be a separate PR once the security agent supported Node 20.